plugins/package-managers/nuget/src/main/kotlin/utils/NuGetUtils.kt

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (C) 2023 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.plugins.packagemanagers.nuget.utils
+
+import org.ossreviewtoolkit.model.Identifier
+
+fun getIdentifierWithNamespace(type: String, name: String, version: String): Identifier {
```

Comment:
Please add KDoc as this is now public. Maybe adding some hint why this unused function exists would  help as well?